force-app/main/default/classes/CarTriggerUtility.cls

```diff
@@ -0,0 +1,63 @@
+public with sharing class CarTriggerUtility {
+    public static void afterInsert(List<Car__c> cars){
+        System.debug('utilitydebug');
+        List<Contact> listOfChangedContacts= new List<Contact>();
+        List<Contact> contacts = new List<Contact>();
+        
+        contacts= [SELECT Id,Expenses__c,LastAddedExpenseValue__c FROM Contact];
+
+        for(Car__c	 car: cars){
+            for(Contact contact: contacts){
+                if(car.Contact__c==contact.id){
+                    System.debug('Found Contact of Car');
+                    contact.LastAddedExpenseValue__c= car.Price__c;
+                    contact.Expenses__c+= car.Price__c;
+                    listOfChangedContacts.add(contact);
+                    }
+            }
+        }
+    
+        update listOfChangedContacts;
+        System.debug('End CarTriggerUtility');
+    
+
+    }
+
+
+
+
+    public static void beforeUpdate(List<Car__c> cars){
+
+    }
+
+    public static void beforeDelete(List<Car__c> cars){
```

Comment:
Troche nie rozumiem tego rozbicia na Profit i Expenses, bo chyba wystarczyłoby, żebyśmy mieli tylko koszty, bo profit to kiedyś mieliśmy robić rozwijając to zadanko i tutaj mieliśmy ograniczyć się do liczenia kosztów i tego, żeby końcowo na accouncie pokazywać jaki był budżet początkowy i ile go zużyto